src/main/CommandHandler.cpp

```diff
@@ -950,8 +926,7 @@ CommandHandler::setcursor(std::string const& params, std::string& retStr)
 
     uint32 cursor;
 
-    if (!parseNumParam(map, "cursor", cursor, retStr,
-                       Requirement::REQUIRED_REQ))
+    if (!parseNumParam(map, "cursor", cursor))
     {
```

Comment:
this code can be deleted

Reply:
Don't we need this check to gracefully handle the case when `cursor` is not passed in the request? With the updated logic based on your comments, it would be something like 
```
    uint32 cursor;
    auto res = parseNumParam(map, "cursor");

    if (!res)
    {
        retStr = "Invalid cursor";
        return;
    }
    else
    {
        cursor = *res;
    }
```

Reply:
If it’s a required parameter this should be taken care of by `parseNumParam`